Sources/CouchDB/Database.swift

```diff
@@ -405,8 +405,37 @@ public class Database {
     }
 
     
-    /// Retrieve all documents from the database
-    ///
+    /**
+     Retrieve all documents in the database using the CouchDB `_all_docs` view.
+     If `includeDocuments` is false, each returned `AllDatabaseDocuments` row will be structured:
```

Comment:
I think the end of this sentence should have the words "as follows:" or "with `id`, `key` and `value` as shown below":